Fix runtime independent example address and port

casey opened the pull request:
I think the address and port in the runtime-independent example are incorrect. They should probably be `0.0.0.0` to listen on all interfaces (or `[::]` for ipv4 and ipv6), and the port should be `443`, since the challenge is always sent to port 443.

FlorianUekermann commented:
I used 4433 to have an example that users can actually run with default user privileges. If you run this on a dev machine you usually need some port forwarding in your router anyway which can map 443 to 4433, similar story for any kind of container. Maybe there should be a comment on the code.

I do agree with changing the ip for the same reason (making the example not crash in a normal dev env). ip v4 and v6 sound good to me.

casey commented:
> I used 4433 to have an example that users can actually run with default user privileges. If you run this on a dev machine you usually need some port forwarding in your router anyway which can map 443 to 4433, similar story for any kind of container. Maybe there should be a comment on the code.

I noticed that there was an unused `Opt::port` field, so I updated the example to use that. How about that as a compromise? It uses the correct port, which users can change if they don't have permissions.

> I do agree with changing the ip for the same reason (making the example not crash in a normal dev env). ip v4 and v6 sound good to me.

Sounds good, I switched it to listen on `::`, which is the IPv6 version of `0.0.0.0`.

FlorianUekermann commented:
This was implemented in October (and made available on crates.io), I just forgot to push everything here. I can't recall now why I didn't use your commits, but I'm fairly sure this pull request was the reason I implemented the changes.
Thanks for your suggestions. Sorry for the spotty communication, Q3 2021 was a very busy period.
